versioned_docs/version-1.0/language/built-in-functions.mdx

```diff
@@ -32,10 +32,17 @@ The message argument is optional.
 ## `revertibleRandom`
 
 ```cadence
-view fun revertibleRandom(): UInt64
+view fun revertibleRandom<T>(modulo: T): T
 ```
+Returns a pseudo-random integer.
+
+`T` can be any fixed-size unsigned integer type
+(`UInt8`, `UInt16`, `UInt128`, `UInt256`, `Word8`, `Word16`, `Word32`, `Word64`).
```

Comment:
```suggestion
view fun revertibleRandom<T: FixedSizeUnsignedInteger>(modulo: T): T
```
Returns a pseudo-random integer.

`T` can be any fixed-size unsigned integer type (`FixedSizeUnsignedInteger`, i.e.
(`UInt8`, `UInt16`, `UInt32`, `UInt64`, `UInt128`, `UInt256`, 
`Word8`, `Word16`, `Word32`, `Word64`, `Word128`, `Word256`).
```